GatherPress/gatherpress
# changed default venue address; removed default address from displaying on the frontend

### Description of the Change
Map with updated default address now displays on backend only (FSE or WP admin page).
Closes #568 

### How to test the Change
Download, create new venue without map address information; navigate to new event and select the new venue with no address. You will see the new default address of WP foundation (660 4th Street San Francisco CA 94107)

### Changelog Entry
> Changed - Default venue map address to WP foundation
> Fixed - Venue map with default address no longer displays on the frontend

### Credits
@stephenerdelyi 

### Checklist:
